plugins/source/aws/resources/services/ecr/repository_images.go

```diff
@@ -54,16 +51,3 @@ func fetchEcrRepositoryImages(ctx context.Context, meta schema.ClientMeta, paren
 	}
 	return nil
 }
-func resolveImageArn(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
```

Comment:
there's no such thing as image ARN according to the [docs](https://docs.aws.amazon.com/service-authorization/latest/reference/list_amazonelasticcontainerregistry.html#amazonelasticcontainerregistry-resources-for-iam-policies)

Reply:
Nice catch! But just so you know ARNs can exist beyond what is supported in IAM policies